### PR TITLE
Added (local)Unifi to ACME actions

### PR DESCRIPTION
**Pull request by adn77**
@fraenki I don't know if this is acceptable to you (regarding the unsupported remark here https://github.com/opnsense/plugins/issues/2627#issuecomment-967663228)

I tested this deployment hook using https://github.com/mimugmail/opn-repo/tree/main/net-mgmt/unifi-maxit

### CODE REVIEW

**Comment by fraenki** [comment]
@adn77 Would you be available if there are issues with this functionality or if someone needs help with it? :)

File: security/acme-client/src/opnsense/mvc/app/library/OPNsense/AcmeClient/LeAutomation/AcmeUnifi.php [hunk 1]
```diff
@@ -0,0 +1,46 @@
+<?php
+
+/*
+ * Copyright (C) 2021 Frank Wall
```
Comment (fraenki) [inline security/acme-client/src/opnsense/mvc/app/library/OPNsense/AcmeClient/LeAutomation/AcmeUnifi.php:4]: I did not create this file. Please remove it and add your name.

**Comment by L1ghtn1ng** [comment]
With unifi OS the keystore if I recall correctly is not used as when I upgraded my cloudkey gen 2 I had to change where the cert went as it was not the keystore anymore. Not sure if that is the case though for having a vm that you setup with just the controller on and not using unifi OS

**Comment by adn77** [comment]
This is for when running Unifi locally (on OPNsense). The Keystore location is a configurable parameter, preset with the location according to this package: https://github.com/mimugmail/opn-repo/tree/main/net-mgmt/unifi-maxit

**Comment by mimugmail** [comment]
If its not acceptable I can also add a certificate field to unifi plugin and place it where you need it

**Comment by adn77** [comment]
Thanks, but no :) The path is already configurable:
grafik

@mimugmail I learned this from your Blog ;)

**Comment by mimugmail** [comment]
The decision is up to @fraenki for me both is ok :)
Seems the guides are not that outdated when you got so far 👍

**Comment by fraenki** [comment]
As stated elsewhere I will not provide support for this, but other than this no objections from my side.
@adn77 agreed to provide support and @mimugmail maintains the unofficial plugin, so everything should be fine :)

**Pull request merged**